_layouts/default2025.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>{{ page.title }}</title>
+    <meta name="description" content="{{ page.description }}">
+    {% if page.stylesheet %}
+    <link rel="stylesheet" href="/assets/css/{{ page.stylesheet }}.css">
+    {% else %}
+    <link rel="stylesheet" href="/assets/css/styles.css">
+    {% endif %}
+    <link rel="icon" href="/assets/img/favicon.png" sizes="192x192">
+    <link rel="icon" href="/assets/img/favicon.ico" sizes="48x48">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    {{ content }}
+    <div class="footerHolder">
+        <div class="container">
+            <footer>
+                <ul class="links">
```

Comment:
Might be nice to use the contents of https://universe.opensuse.org/api/v0/sites.json

Reply:
I'll need to check if there's an easy way to do this in Liquid syntax.

Reply:
Liquid syntax probably not, but if you want I can write a quick plugin to make this data available to you through the site variables

Reply:
There you go, it's quick and dirty but it does the job:
https://gist.github.com/hellcp/a02a40d338bf72290247495239289953

Reply:
Thanks, that works! I've also replaced the static social media links with the ones from sites.json.

Reply:
I've reworked the footer a little bit:

![Screen Shot 2025-05-10 at 22 40 52](https://github.com/user-attachments/assets/b8e445c0-2d7d-46ed-a664-911a8079ae3d)

The footer link layout is a little bit more spacious than the previous menu layout to make it easier to tap the links on mobile screens.